modules/accounts/src/lib.rs

```diff
@@ -283,6 +302,9 @@ impl<T: Trait> Module<T> {
 			// by treasury account is sufficient to open an account.
 			if *k != treasury_account {
 				// send dust native currency to treasury account
+				// REVIEW: IIUC the account could have lots of reserved balance, just not any
+				//         free balance and thus should maybe not be considered dust just based
+				//         on the reserving failing?
 				let _ = <T as Trait>::Currency::transfer(
```

Comment:
Addressed: `open_account` is a private function that is only ever called with new accounts so they won't have reserved balance.